doc/adr/0003-graphql-as-api-query-language-via-appolo-server.md

```diff
@@ -0,0 +1,21 @@
+# 3. GraphQL as API Query Language via Apollo Server
+
+Date: 2020-06-06
+
+## Status
+
+Accepted
+
+## Context
+
+We need a way to populate our Neo4J database via an API layer.
+
+## Decision
+
+Use GraphQL libraries for the server API. GraphQL has been increasing in popularity and maturity lately. It provides a more flexible API query layer compared to REST based interactions.
```

Comment:
An extra benefit is we're already doing arbitrarily structured queries, with optional sub-elements and with query parameters.  This is where GraphQL really shines, compared to traditional REST query endpoints.

Reply:
I think that what I was glossing over what I said 'more flexible'. 
Will add that detail in.